src/components/Connection/Desktop/CreateAccount.vue

```diff
@@ -0,0 +1,487 @@
+          <s-input
+            :type="inputType"
+            :placeholder="t('desktop.password.placeholder')"
+            v-model="accountPassword"
+            :disabled="loading"
+          >
+            <s-icon
+              :name="iconPasswordStyle"
+              size="18"
+              class="eye-icon"
+              slot="suffix"
+              @click.native="toggleVisibility"
+            />
+          </s-input>
+          <p class="login__create-account-desc">{{ t('desktop.password.desc') }}</p>
+          <s-input
+            type="password"
+            :placeholder="t('desktop.confirmPassword.placeholder')"
+            v-model="accountPasswordConfirm"
+            :disabled="loading"
+          ></s-input>
+        </s-form>
+      </div>
+      <div class="wallet-settings-create-token_export">
+        <s-switch v-model="toExport" :disabled="loading" />
+        <span>{{ t('desktop.exportOptionText') }}</span>
+      </div>
+      <p class="wallet-settings-create-token_desc">{{ t('desktop.exportJsonText') }}</p>
+      <a ref="json" class="download-json" />
+      <s-button
+        key="step3"
+        @click="createAccount"
+        class="s-typography-button--large login-btn"
+        type="primary"
+        :disabled="isInputsNotFilled"
+      >
+        {{ t('desktop.button.createAccount') }}
+      </s-button>
+    </template>
+  </div>
+</template>
+
+<script lang="ts">
+import { Mixins, Component, Prop, Ref } from 'vue-property-decorator';
+import { isEqual } from 'lodash';
+
+import LoadingMixin from '../../../components/mixins/LoadingMixin';
+import TranslationMixin from '../../../components/mixins/TranslationMixin';
+import { api } from '../../../api';
+import { LoginStep } from '../../../consts';
+import { copyToClipboard } from '../../../util';
+import { action } from '../../../store/decorators';
+
+@Component
+export default class CreateAccount extends Mixins(TranslationMixin, LoadingMixin) {
+  @action.account.getPolkadotJsAccounts getPolkadotJsAccounts!: () => Promise<void>;
+
+  @Prop({ type: String }) readonly step!: LoginStep;
+  @Ref('json') readonly json!: HTMLLinkElement;
+
+  readonly LoginStep = LoginStep;
+  readonly PHRASE_LENGTH = 12;
+
+  accountName = '';
+  accountPassword = '';
+  accountPasswordConfirm = '';
+  fileName = '';
+  href: string | null = null;
+
+  seedPhraseToCompare: Array<string | undefined> = [];
+  seedPhraseBoundToClass = new Map<string | undefined, string>();
+
+  showErrorMessage = false;
+  hiddenInput = true;
+  toExport = false;
+
+  get title(): string {
+    if (this.step === LoginStep.SeedPhrase) return this.t('desktop.heading.seedPhraseTitle');
+    if (this.step === LoginStep.ConfirmSeedPhrase) return this.t('desktop.heading.confirmSeedTitle');
+    if (this.step === LoginStep.CreateCredentials) return this.t('desktop.heading.accountDetailsTitle');
+    return '';
+  }
+
+  get stepNumber(): number {
+    if (this.step === LoginStep.SeedPhrase) return 1;
```

Comment:
enum LoginStep might contain numbers
```
enum LoginStep {
  SeedPhrase = 1,
  ConfirmSeedPhrase = 2,
  CreateCredentials = 3
}
```
`stepNumber` it might be removed

Reply:
LoginStep contains a little bit more complicated structure as we have import account option, entering by seed phrase, and creating a new one from scratch.
<img width="529" alt="Screenshot 2022-09-06 at 10 40 19" src="https://user-images.githubusercontent.com/27702061/188576298-3b2e63b7-c247-4448-9392-81c423328ba8.png">

BTW, these digits are rendered here
<img width="322" alt="Screenshot 2022-09-06 at 10 44 53" src="https://user-images.githubusercontent.com/27702061/188576642-cc2f9f58-802f-4e89-90a9-c88c791e2d2d.png">
<img width="336" alt="Screenshot 2022-09-06 at 10 44 46" src="https://user-images.githubusercontent.com/27702061/188576648-8a7e0e2b-b64c-4d60-9591-9a11ddcb6928.png">